common/index.ts

```diff
@@ -44,6 +44,8 @@ export const MAX_LAYER_NAME_LIMIT = 35;
 export const MAX_LONGITUDE = 180;
 export const MIN_LONGITUDE = -180;
 export const NEW_MAP_LAYER_DEFAULT_PREFIX = 'New layer';
+export const MAP_SAVED_OBJECT_TYPE = 'map';
+export const MAPS_APP_ICON = 'globe';
```

Comment:
@kgcreative  We need new icon for Maps. 
 @junqiu-lei Can you add TO DO and create github issue to replace icon?

Reply:
Updated

Reply:
Created a issue to follow up: https://github.com/opensearch-project/oui/issues/292